### PR TITLE
Draft dependencies/removals article for Chrome 60.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: src/content/en/updates/2017/03/chrome-58-deprecations.md [hunk 1]
```diff
@@ -53,6 +53,21 @@ applied. The old behavior was deprecated in Chrome 57, and is now removed.
 [Chromestatus Tracker](https://www.chromestatus.com/feature/5760965337415680) &#124;
 [Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=659464)
 
+## Remove content-initiated top frame navigations to data URLs
+
+Because of their unfamilliarity to non-technical browser users, we're
```
Comment (meacer) [inline src/content/en/updates/2017/03/chrome-58-deprecations.md]: "unfamilliarity" spelling

File: src/content/en/updates/2017/03/chrome-58-deprecations.md [hunk 2]
```diff
@@ -53,6 +53,21 @@ applied. The old behavior was deprecated in Chrome 57, and is now removed.
...
+increasingly seeing the `data:` scheme being used in spoofing and phishing
+attacks. To prevent this, we're blocking web pages from loading using `data:`
+from the top frame. This applies to `&lt;a&gt;` tags, `window.open`,
```
Comment (meacer) [inline src/content/en/updates/2017/03/chrome-58-deprecations.md]: I suggest rephrasing this as "we're blocking web pages from loading data: URLs in the top frame".

File: src/content/en/updates/2017/06/chrome-60-deprecations.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+resources loaded below by a page.
+
+This feature was deprecated in Chrome 58 and is now removed.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/GbVcuwg_QjM/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5669602927312896) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=684011&desc=2)
+
+
+### Temporarily remove navigator.setBeacon()
```
Comment (tyoshino) [inline src/content/en/updates/2017/06/chrome-60-deprecations.md]: set -> send

File: src/content/en/updates/2017/06/chrome-60-deprecations.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+The `navigator.setBeacon()` function has been available
```
Comment (tyoshino) [inline src/content/en/updates/2017/06/chrome-60-deprecations.md]: set -> send

File: src/content/en/updates/2017/06/chrome-60-deprecations.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+[since Chrome 39](https://www.chromestatus.com/feature/5517433905348608). 
+As originally implemented, the function's `data` argument could contain any
+arbitrary blob whose type is not CORS-safelisted. We believe this is a potential
+security thread, though no on has yet tried to exploit it. Because we do NOT
```
Comment (tyoshino) [inline src/content/en/updates/2017/06/chrome-60-deprecations.md]: thread -> threat?

Comment (tyoshino) [inline src/content/en/updates/2017/06/chrome-60-deprecations.md]: no on -> no one?

File: src/content/en/updates/2017/06/chrome-60-deprecations.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+have a reasonable immediate fix for it, the `sendBeacon()` function has been
+disabled until further notice.
```
Comment (tyoshino) [inline src/content/en/updates/2017/06/chrome-60-deprecations.md]: (disabled only when invoked with a blob whose type is not CORS-safelisted)

File: src/content/en/updates/2017/06/chrome-60-deprecations.md (hunk 3 above)
Comment (tyoshino) [inline src/content/en/updates/2017/06/chrome-60-deprecations.md]: Temporarily disable navigator.sendBeacon() for a particular kind of arguments

**Comment by tyoshino** [comment]
Looks good. Please reflect the change to M59 message. Thanks!

**Comment by jpmedley** [comment]
Good catch. Done.

**Comment by tyoshino** [comment]
d94c6ada9c536fe7275f7680596858d2e4e11163 looks good for sendBeacon. thanks!

**Pull request merged**